standard/expressions.md

```diff
@@ -1025,6 +1071,19 @@ In case the parameter type sequences `{P₁, P₂, ..., Pᵥ}` and `{Q₁, Q₂
 - Otherwise if one member is a non-lifted operator and the other is a lifted operator, the non-lifted one is better.
 - If neither function member was found to be better, and all parameters of `Mᵥ` have a corresponding argument whereas default arguments need to be substituted for at least one optional parameter in `Mₓ`, then `Mᵥ` is better than `Mₓ`. Otherwise, no function member is better.
```

Comment:
@BillWagner: While editing these rules, can we put "Otherwise, no function member is better" in its own bullet. (Only if this set of rules does get edited.)

Reply:
New rule added, and final "otherwise" in its own bullet.